pkg/helm/handlers/handlers.go

```diff
@@ -438,3 +441,28 @@ func (h *helmHandlers) HandleIndexFile(user *auth.User, w http.ResponseWriter, r
 
 	w.Write(out)
 }
+
+func (h *helmHandlers) HandleUninstallReleaseAsync(user *auth.User, w http.ResponseWriter, r *http.Request) {
+	params := r.URL.Query()
+	ns := params.Get("ns")
+	rel := params.Get("name")
+	version := params.Get("version")
+	conf := h.getActionConfigurations(h.ApiServerHost, ns, user.Token, &h.Transport)
+	coreClient, err := NewCoreClient(conf)
+	if err != nil {
+		serverutils.SendResponse(w, http.StatusBadRequest, serverutils.ApiError{Err: fmt.Sprintf("error forming core client: %s", err.Error())})
+		return
+	}
+	resp, err := h.uninstallReleaseAsync(rel, ns, version, conf, coreClient)
+	if err != nil {
+		if err.Error() == actions.ErrReleaseNotFound.Error() {
+			serverutils.SendResponse(w, http.StatusNotFound, serverutils.ApiError{Err: fmt.Sprintf("Failed to uninstall helm release: %v", err)})
+			return
+		}
+		serverutils.SendResponse(w, http.StatusBadGateway, serverutils.ApiError{Err: fmt.Sprintf("Failed to uninstall helm release: %v", err)})
+		return
+	}
+	w.Header().Set("Content-Type", "application/json")
+	res, _ := json.Marshal(resp)
+	w.Write(res)
```

Comment:
I believe there is no benefit from returning the Secret as JSON here. We just need a confirmation:

```suggestion
w.WriteHeader(http.StatusNoContent)
```

Reply:
I think this makes sense.Thanks.